### PR TITLE
fixed enviroment variable false positive

### PR DESCRIPTION
**Pull request by villav**
from Chicago PHP Open Source workshop.

### CODE REVIEW

**Comment by SammyK** [comment]
:+1:

**Comment by gfosco** [comment]
Hi, Can you please sign the CLA?

**Comment by villav** [comment]
Signed

File: tests/FacebookTest.php [hunk 1]
```diff
@@ -34,6 +34,7 @@
 use Facebook\Authentication\AccessToken;
 use Facebook\GraphNodes\GraphList;
 
+
```
Comment (yguedidi) [inline tests/FacebookTest.php]: useless line

**Pull request merged**